test/greet-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const Greet = require('../lib/greet.js');
+const expect = require('chai').expect;
+
+describe('Greet Module Test', function(){
+  describe('#greet', function() {
```

Comment:
Good test. Would like to see another test here of you testing the method NOT working properly.